client/app/dedicated/server/dedicated-server.html

```diff
@@ -27,13 +27,15 @@
                                   data-disabled="server.isExpired"></oui-header-tabs-item>
             <oui-header-tabs-item data-text="{{:: 'server_tab_ftp_backup' | translate }}"
                                   data-state="app.dedicated.server.ftpBackup"
-                                  data-disabled="server.isExpired"></oui-header-tabs-item>
+                                  data-disabled="server.isExpired"
+                                  data-ng-if="worldPart !== 'US'"></oui-header-tabs-item>
```

Comment:
we should probably uniformize with `coreConfig.getRegion()`

Reply:
This is the case in the associated controller:
https://github.com/ovh-ux/ovh-manager-dedicated/blob/7ec657111e2abc5aaa064e580d0c33abe93965df/client/app/dedicated/server/dedicated-server.controller.js#L42